### ISSUE TITLE
Toggle Landing Gear keybind overrides No Gearup on Ground and allows gear lever to be moved

### ISSUE THREAD

**Issue by 2hwk** [flybywiresim/a32nx#5874]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-09-19T20:59:10+00:00",
  "ref": "master",
  "sha": "74045fb5eb77b3043f816d8d26028c33fb63bfd4",
  "actor": "davidwalschots",
  "event_name": "manual"
}
```

### Describe the bug

When using the mouse to move the landing gear lever up, it behaves as it should, where this is disabled on ground. 

However, using a keybind set to the ``KEY_GEAR_TOGGLE`` keyboard event allows the gear lever to be manipulated, and it ignores the default sim configuration that is supposed to manage this.

### Expected behavior

TOGGLE GEAR Keybind should not allow you to move the gear lever if you are unable to do so with the mouse.

### Steps to reproduce

1. Spawn in cold/dark or on the ground state
2. Try to move the gear lever with the mouse, the gear lever should not move
3. Use a keybind to move the gear lever (set Toggle Gear in keybinds). The gear lever moves when it should not.

### Discord Username (optional)

2Cas#1022

**Comment by davidwalschots** [comment]
I'm assigning myself to this as #6427 might make it very simple to fix this behaviour soon.

**Comment by derl30n** [comment]
@davidwalschots has your pr fixed this issue or is it still persistent?

**Comment by derl30n** [comment]
Issue persists.

**Comment by davidwalschots** [comment]
Sorry about that. The PR wasn't intended to fix this issue, but likely has made it easier to fix this issue in a Rusty context. However, I haven't spent the time to fix this. Removed myself from this bug.

**Comment by crocket63** [comment]
Will be solved in #6893 !
